packages/zambdas/src/ehr/get-lab-orders/helpers.ts

```diff
@@ -810,7 +810,7 @@ export const fetchFinalAndPrelimAndCorrectedTasks = async (
   oystehr: Oystehr,
   results: DiagnosticReport[]
 ): Promise<Task[]> => {
-  const resultsIds = results.map((result) => result.id).filter(Boolean);
+  const resultsIds = results.map((result) => `DiagnosticReport/${result.id}`).filter(Boolean);
```

Comment:
Looks like filter(Boolean) doesn't work here, because if result.id is undefined after map, there will be a string value. If filter is needed, it should be before the map. Or we can use reduce instead of map.

```suggestion
  const resultsIds = results.reduce(
    (acc, result) => (result?.id ? [...acc, `DiagnosticReport/${result.id}`] : acc),
    [] as string[]
  );
```

Reply:
Would i diagnostic report ever not have an Id ?

Reply:
actually i guess by that argument, do we even need the filter(Boolean)?

Reply:
Typing seems fine for both 
```const resultsIds = results.map((result) => `DiagnosticReport/${result.id}`);```
and 
```const resultsIds = results.map((result) => result.id);

But its fine, i'll just update it to the reduce as you suggest.